src/app/scenes/email-registration/email-registration/email-registration.component.html

```diff
@@ -0,0 +1,52 @@
+<div>
+    <mat-progress-spinner [mode]="'indeterminate'" *ngIf="loading$ | async"></mat-progress-spinner>
+    <div *ngIf="!(loading$ | async)">
+        <div class="util--backdrop modal-backdrop" *ngIf="modalOpen" (click)="closeModal()">
+            <div class="confirmation-modal">
+                <div class="close-icon" (click)="closeModal()"><mat-icon>close</mat-icon></div>
+                <div class="modal__title">{{ 'EmailRegistration.ConfirmationTitle' | translate }}</div>
+                <div class="modal__text">{{ 'EmailRegistration.ConfirmationText' | translate }}</div>
+                <button class="button modal-button"
+                (click)="closeModal()">{{ 'EmailRegistration.ConfirmationBtnText' | translate }}</button>
+                <button (click)="routeToPayment()" class="button button--black modal-button">{{ 'EmailRegistration.ConfirmationBtnText2' | translate }}</button>
```

Comment:
i didn't run the code yet, but why is routeToPayment() unsetting the email? How do you get in this path?

Reply:
oh wait.. that's the skip step?

ConfirmationbtnText2 isn't very useful name :D I need to go to the translations to know what it means. Can you change to something like ConfirmationBtnContinueWithout ?